mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -183,7 +165,7 @@ def get_output_args(self, target: str) -> T.List[str]:
         return ['/Fo' + target]
 
     def get_buildtype_args(self, buildtype: str) -> T.List[str]:
-        return msvc_buildtype_args[buildtype]
+        return []
```

Comment:
is override of the base method required at all now?

Reply:
Yes, if it is not implemented it will throw an exception.

Reply:
cool